src/main/java/racingcar/view/dto/TrialTimes.java

```diff
@@ -0,0 +1,51 @@
+package racingcar.view.dto;
+
+import racingcar.util.ErrorMessage;
+
+public class TrialTimes {
+    private static final int TRIAL_MAX_TIMES = 100;
+    private static final int TRIAL_MIN_TIMES = 1;
+
+    private final Integer trialTimes;
+
+    public TrialTimes(String trialTimes) {
+        validate(trialTimes);
+        this.trialTimes = Integer.parseInt(trialTimes);
+    }
+
+    private void validate(String trialTimes) {
+        validateNumber(trialTimes);
+        validateRange(trialTimes);
+    }
+
+    private void validateNumber(String trialTimes) {
+        try {
+            Integer.parseInt(trialTimes);
+        } catch (NumberFormatException exception) {
+            throw new IllegalArgumentException(ErrorMessage.INVALID_TRIAL_NUMBER.message());
+        }
+    }
+    
+    private void validateRange(String trialTimes) {
+        int trialTimesNumber = Integer.parseInt(trialTimes);
+
+        isOverMaxRange(trialTimesNumber);
+        isUnderMinRange(trialTimesNumber);
+    }
+
+    private void isOverMaxRange(int trialTimesNumber) {
+        if (trialTimesNumber > TRIAL_MAX_TIMES) {
+            throw new IllegalArgumentException(ErrorMessage.TRIAL_NUMBER_OVER_RANGE.message());
+        }
+    }
+
+    private void isUnderMinRange(int trialTimesNumber) {
+        if (trialTimesNumber < TRIAL_MIN_TIMES) {
+            throw new IllegalArgumentException(ErrorMessage.TRIAL_NUMBER_UNDER_RANGE.message());
+        }
+    }
```

Comment:
boolean을 return하는 것이 아니라면 is가 아닌 validate로 함수명을 가져가보는 건 아떨까요?
```suggestion
    private void validateOverMaxRange(int trialTimesNumber) {
        if (trialTimesNumber > TRIAL_MAX_TIMES) {
            throw new IllegalArgumentException(ErrorMessage.TRIAL_NUMBER_OVER_RANGE.message());
        }
    }

    private void validateUnderMinRange(int trialTimesNumber) {
        if (trialTimesNumber < TRIAL_MIN_TIMES) {
            throw new IllegalArgumentException(ErrorMessage.TRIAL_NUMBER_UNDER_RANGE.message());
        }
    }
```